src/index.js

```diff
@@ -4,13 +4,26 @@
 
 // Koa application to provide HTTP interface.
 
+const fs = require("fs");
+const envPaths = require("env-paths");
+const config_file =
+  envPaths("ssb-oasis", { suffix: "" }).config + "/config.json";
```

Comment:
Found the reason `~/.config/oasis` wasn't working!

Reply:
yeah. I wanted to just use oasis, but on debian/ubuntu that's already an application that seems quite a bit older. I don't know if they use ~/.config/oasis but I didn't feel like pushing it...

Reply:
okay I'm back from the dead. really needed a good... 6h30 nap. :D
okay, I [searched for XDG_CONFIG_HOME in the other oasis' repo](https://github.com/ocaml/oasis/search?q=XDG_CONFIG_HOME&unscoped_q=XDG_CONFIG_HOME) and came up empty. I'll force-push the version with just ~/.config/oasis then. :)